stable/coscale/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for CoScale Helm package.
+
+image:
+    repository: "coscale/coscale-agent"
+    tag: "latest"
+    pullPolicy: "Always"
+
+coscale:
+    # Required: You need a CoScale AppId before running agents.
+    AppId: ""
+
+    # Required: You need a CoScale AccessToken before running agents.
+    AccessToken: ""
+
+    # Required: You need a CoScale TemplateId before running agents.
+    TemplateId: ""
+
+resources:
```

Comment:
We've started to recommend not specifying default resources. This should be a conscious choice of the user.